### PR TITLE
Update magenta-install.sh

### PR DESCRIPTION
**Pull request by falaktheoptimist**
As per #1477 anaconda default changed to python3

### CODE REVIEW

File: magenta/tools/magenta-install.sh [hunk 1]
```diff
@@ -85,7 +85,7 @@ echo "setting up magenta environment"
 echo "=============================="
 echo ""
 
-conda create -n magenta python=2.7
+conda create -n magenta python=3.6
```
Comment (cghawthorne) [inline magenta/tools/magenta-install.sh]: should probably also change the download above to get Miniconda3 instead of Miniconda2.

Comment (falaktheoptimist) [reply magenta/tools/magenta-install.sh]: Done. Also, pip changed to pip3.

Comment (cghawthorne) [reply magenta/tools/magenta-install.sh]: one other nit: can you change the path for CONDA_PREFIX to also use miniconda3?

Comment (falaktheoptimist) [reply magenta/tools/magenta-install.sh]: Done

Comment (cghawthorne) [inline magenta/tools/magenta-install.sh]: Let's actually move this to python=3.7 since that's what we do our Travis tests with

Comment (falaktheoptimist) [reply magenta/tools/magenta-install.sh]: Done.

**Pull request merged**